packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressSearchSuggestions.vue

```diff
@@ -1,15 +1,22 @@
 <template>
-    <div :class="$style.shell">
+    <div
+        ref="fullAddressSuggestions"
+        :class="$style.shell">
         <button
             v-for="(item, index) in getAddressItems"
-            :key="`${index}_ getAddressItem`"
-            type="button"
```

Comment:
Do you not need this attribute anymore?

Reply:
I may need it soon but have removed it for now. Been having an issue with the enter key when using the keyboard and I thought setting it to a type of button would allow programmatic selection of address results, wasn't working (works ok in the current version) need to investigate it 😄